appium/api-mocks/mock-data.ts

```diff
@@ -2,6 +2,7 @@ export type MockUser = {
   email: string;
   name: string;
   pub?: string;
+  pubNew?: string;
   pubOther?: string;
```

Comment:
what about merging these properties in a single `pubs` array?
so you won't need to find another name when we'll need to have 4 different pubs here :)

Reply:
Well, but if we do like that we would lose key meanings.
Like `pubNew` is newer version of `pub` key.
If we combine them into `pubs` array, devs won't have idea which is newer/older version key.

Of course, we can add PubKey interface something like `{ new?: string, old?: string}` and defined pubs as PubKey[] but it would give us complexness I think.

Reply:
Okay, let's leave it for now